.github/workflows/ci.yml

```diff
@@ -35,13 +35,12 @@ jobs:
           apt-get update
           apt-get install -y curl git
 
-      - name: Install Magic CLI
+      - name: Install Pixi
         shell: bash
         run: |
-          curl -ssL https://magic.modular.com/ | bash
-          . /root/.bashrc
-          echo "/root/.modular/bin" >> $GITHUB_PATH
-          export PATH="/root/.modular/bin:$PATH"
+          curl -fsSL https://pixi.sh/install.sh | PIXI_VERSION=0.47.0 sh
```

Comment:
That'll do and simpler is with https://github.com/modular/mojo-gpu-puzzles/blob/main/.github/workflows/ci.yml#L37

Reply:
Done